release-info/calendar.md

```diff
@@ -0,0 +1,27 @@
+# Prospective release calendar
+
+This is a prospective calendar for the next releases of OCaml.
+
+This document is intended to give a very rough idea of the timeline for the next
+versions of OCaml to anyone interested. However, it would be an unforeseen
+accident if this prospective calendar ever matches the real release calendar.
+
+
+# Main versions
+
+## OCaml 5.3.0
+
+- Feature freeze and alpha release: 15th August 2024
+- First beta release: [10th September, 15th October]
```

Comment:
I don't quite uinderstand why there's a pair of dates. Maybe an explaining sentence would make sense?

Reply:
This is a good idea: I am using a range of date in order to try convey the uncertainty over those dates in a slightly quantified way.

Reply:
Square brackets in md can be meaningful on their own (depending on the flavour you use) so that's confusing too.

Reply:
I'm likely very biased, but I find the FreeBSD target release schedule has a very nice overview, take a look at https://www.freebsd.org/releases/14.1R/schedule/

So, they (a) use a table, (b) have a expected and actual date. The website linked above is linked from the main one https://www.freebsd.org/releng/ that contains the planned releases.

It may as well make sense to embed a "last updated" to the `calendar.md`, to give an impression how much up-to-date that information is. :)

Thank you for working on documentation of the release process! \o/

Reply:
A table like

|    Phase              | Expected (early) | Expected (late) | Actual        |
|-----------------------|------------------|-----------------|---------------|
| Feature freeze        | 15 August 2024   | (same)          |               |
| 1st Beta release      | 10th September   | 15th October    |               |
| 1st release candidate | 1st October      | 7th November    |               |
| Release               | 7th October      | 21st November   |               |


 sounds like a good idea, thanks for the suggestion.